src/components/SearchField.tsx

```diff
@@ -87,28 +89,37 @@ const SearchField: React.FC = () => {
   const fromDateInputIncrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.fromYear >= dateRange.toYear) return;
+    if (dateRange.fromYear >= dateRange.toYear) {
+      setIsDataValid(true);
+      setTimeout(() => setIsDataValid(false), 2000);
+      return;
+    }
     setDateRange({ ...dateRange, fromYear: dateRange.fromYear + 1 });
   };
 
   const fromDateInputDecrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.fromYear <= lowerDateLimit) return;
+    setIsDataValid(false);
     setDateRange({ ...dateRange, fromYear: dateRange.fromYear - 1 });
   };
 
-  const toYearInputIncrease = (
+  const toDateInputIncrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
     const upperDataLimit = new Date().getFullYear();
     if (dateRange.toYear >= upperDataLimit) return;
+    setIsDataValid(false);
     setDateRange({ ...dateRange, toYear: dateRange.toYear + 1 });
   };
-  const toYearInputDecrease = (
+  const toDateInputDecrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.toYear <= dateRange.fromYear) return;
+    if (dateRange.toYear <= dateRange.fromYear) {
+      setIsDataValid(true);
```

Comment:
same as above

Reply:
Here changing a state allows handling a message appropriately (make it appear on a screen when a condition met)